lib/Conch/Controller/DB/HardwareProduct.pm

```diff
@@ -1,7 +1,10 @@
 package Conch::Controller::DB::HardwareProduct;
 
+use Role::Tiny;
```

Comment:
minor quibble: this should be Role::Tiny::With. (`use Role::Tiny` makes this module into a role.)  We don't want our controllers to be roles :)

Reply:
done